int64 grasp_id
int64 object_id
string object_name
float64 time_stamp
geometry_msgs/PoseStamped preshape_palm_world_pose
geometry_msgs/PoseStamped true_preshape_palm_world_pose
sensor_msgs/JointState preshape_allegro_joint_state
sensor_msgs/JointState true_preshape_joint_state
sensor_msgs/JointState close_shape_allegro_joint_state
geometry_msgs/PoseStamped close_shape_palm_world_pose
sensor_msgs/JointState lift_shape_allegro_joint_state
geometry_msgs/PoseStamped lift_shape_palm_world_pose
# Object pose computed from segmentation
geometry_msgs/PoseStamped object_world_seg_pose
# Object pose of object spawning in simulation
# geometry_msgs/PoseStamped object_world_sim_pose
int32 grasp_success_label
bool top_grasp
int32[] sparse_voxel_grid
float32[] object_size
float32[] inf_ik_config_array
float32[] inf_config_array
float32 inf_val
float32 inf_suc_prob
float32 inf_log_prior
float32 inf_uct
float32 reward
string action
float32[] init_ik_config_array
float32[] init_config_array
float32 init_val
float32 init_suc_prob
float32 init_log_prior
float32 init_uct
string prior_name
---
bool save_h5_success
# int64 object_id
